### ISSUE TITLE
some question about face training 

### ISSUE THREAD

**Issue by bayminbx** [SystemErrorWang/CartoonGAN#13]
when I train cartoongan(use face images and real_world images), some strange color appear on the face. And in cartoon face images, big eyes are usual.Can you give me some advice?

**Comment by SystemErrorWang** [comment]
@bayminbx the results you see here are generated with some other model (not CartoonGAN in fact), the paper will appear on CVPR 2020 and the code and pre-trained model will be released later.

**Comment by SystemErrorWang** [comment]
@bayminbx As for the problem you mentioned, I also came across when I was training the original CartoonGAN. Maybe you can try increasing the weight of content loss.

**Comment by bayminbx** [comment]
hi, how do you solve the "multi scale" problem? I train the network with size 256 face images, but in inference the size of images is 512. The result is bad. Can you give me some advice?Thank you.

**Comment by SystemErrorWang** [comment]
New model is open-sourced， maybe you can try that one:
https://github.com/SystemErrorWang/White-box-Cartoonization
